internal/server/data/selectors.go

```diff
@@ -119,8 +121,13 @@ func ByUserID(userID uid.ID) SelectorFunc {
 	}
 }
 
-func ByPagination(pg models.Pagination) SelectorFunc {
+func ByNotExpired() SelectorFunc {
+	return func(db *gorm.DB) *gorm.DB {
+		return db.Where("expires_at is null or expires_at > ?", time.Now().UTC()).Or("expires_at is ?", time.Time{})
```

Comment:
Isn't the `OR` in here twice? Once in the first statement and again because of the `.Or(...)` ?

Reply:
these are three different checks: expiration, null, and UTC 0. I can change all of them to `.Or` for consistency.